packages/app/components/Queue/StudentQueueList.tsx

```diff
@@ -218,11 +218,16 @@ export default function StudentQueueList({
           />
           <VerticalDivider />
           <Space direction="vertical" size={40} style={{ flexGrow: 1 }}>
-            <StudentBanner
-              queueId={qid}
-              editQuestion={openEditModal}
-              leaveQueue={leaveQueue}
-            />
+            {studentQuestion &&
```

Comment:
@liustanley what's the point of this? Isn't the conditional render already handled inside studentbanner?

Reply:
I added that because `StudentBanner` still returns a DOM object, so the `Space` component adds unnecessary padding above the Queue header when a student question isn't present. I'm realizing now though that I only have to check for studentQuestion so I'll update that.